process/preprocess/eventsPreProcessorV0.go

```diff
@@ -114,6 +94,59 @@ func (d *eventsPreProcessorV0) setTransactionsPool(txsPool *data.TransactionsPoo
 	}, nil
 }
 
+func (d *eventsPreProcessorV0) parseTxs(txs map[string]*data.NodeTransaction) map[string]*outport.TxInfo {
+	newTxs := make(map[string]*outport.TxInfo, len(txs))
+
+	for hash, txHandler := range txs {
+		if txHandler == nil {
+			continue
```

Comment:
should we continue here or return error? if we continue, then L98 should be `newTxs := make(map[string]*outport.TxInfo, len(txs))` to avoid using extra space

Reply:
yes, it's fine to continue here